derrida/books/management/commands/reference_data.py

```diff
@@ -0,0 +1,94 @@
+import codecs
+import csv
+import json
+import os.path
+
+from django.core.management.base import BaseCommand
+
+from derrida.books.models import DerridaWork
+
+
+class Command(BaseCommand):
+    '''Export reference data for each Derrida Work as CSV and JSON'''
+    help = __doc__
+
+    #: fields for CSV output
+    csv_fields = [
+        'page', 'page location', 'book title', 'book id',
+        'book page', 'type',
+        'anchor text', 'interventions'
+    ]
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-d', '--directory',
```

Comment:
can/does this default to cwd?

Reply:
Yes, defaults to current directory and the argument is optional. Should I clarify in help text?

Reply:
sure, wouldn't hurt